### ISSUE TITLE
Status of artifacts and Scala 3

### ISSUE THREAD

**Issue by Sciss** [velvia/msgpack4s#17]
Hi there, seems the artifacts can no longer be found. Would it be possible to publish them to Maven Central? Does this library work with Scala 3 as well?

**Comment by velvia** [comment]
Yes sorry I’ll try to publish new ones soon.   Need to find a different place to publish them.
